packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -1,84 +1,161 @@
 import './WalletCashierHeader.scss';
 
-const realAccountTabs = ['deposit', 'withdraw', 'transfer', 'transactions'];
-const virtualAccountTabs = ['withdraw', 'transfer', 'transactions', 'reset-balance'];
+type TProps = {
+    hideWalletDetails: boolean;
+};
+
+const realAccountTabs = [
+    {
+        icon: <DepositPlus />,
+        path: 'deposit',
+        text: 'Deposit',
+    },
+    {
+        icon: <WithdrawMinus />,
+        path: 'withdraw',
+        text: 'Withdraw',
+    },
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+] as const;
 
-const WalletCashierHeader = ({ hideWalletDetails }: { hideWalletDetails: boolean }) => {
-    const { data } = useActiveWalletAccount();
+const virtualAccountTabs = [
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+    {
+        icon: <DepositPlus />,
+        path: 'reset-balance',
+        text: 'Reset balance',
+    },
+] as const;
+
+const WalletCashierHeader: React.FC<TProps> = ({ hideWalletDetails }) => {
+    const { data: activeWallet } = useActiveWalletAccount();
     const { isMobile } = useDevice();
+    const activeTabRef = useRef<HTMLButtonElement>(null);
     const history = useHistory();
     const location = useLocation();
+    const tabs = activeWallet?.is_virtual ? virtualAccountTabs : realAccountTabs;
 
-    let tabs = realAccountTabs;
-    if (data?.is_virtual) {
-        tabs = virtualAccountTabs;
-    }
+    useEffect(() => {
+        if (isMobile && activeTabRef.current) {
+            activeTabRef.current.scrollIntoView({ behavior: 'smooth', block: 'end', inline: 'center' });
+        }
+    }, [location.pathname, isMobile]);
 
     return (
         <WalletGradientBackground
-            currency={data?.currency_config?.display_code || 'USD'}
+            currency={activeWallet?.currency_config?.display_code || 'USD'}
             device={isMobile ? 'mobile' : 'desktop'}
+            isDemo={activeWallet?.is_virtual}
             theme='light'
             type='header'
         >
             <main className='wallets-cashier-header'>
                 <section className='wallets-cashier-header__info'>
-                    <div className='wallets-cashier-header__info__top-left'>
+                    <div className='wallets-cashier-header__top-left-info'>
                         <div
-                            className={classNames('wallets-cashier-header__info__top-left__details', {
+                            className={classNames('wallets-cashier-header__details', {
                                 'wallets-cashier-header__hide-details': isMobile && hideWalletDetails,
```

Comment:
I know it was already introduced before this PR, but it caught my attention - __hide-details looks like a modifier, not block according to BEM standards.

Reply:
oh, @wojciech-deriv yeap, totaly agree with you